include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -134,7 +134,7 @@ namespace parallel
       virtual ~Triangulation() = default;
 
       /**
-       * @copydoc dealii::Triangulation::create_triangulation()
+       * @copydoc deal.II/grid/tria.h
```

Comment:
I don't think this is right. From my reading of the doxygen manual, this copies the documentation attached to the *file* `tria.h`, but you want the documentation of a specific member function.

I suspect that doxygen's complaint is that there are two overloaded versions of the function, and it can't figure out which one we are referencing without providing the argument list. As https://www.doxygen.nl/manual/commands.html#cmdcopydoc points out, one can specify the argument list.

Reply:
Well, that's a good enough reason! Good luck tomorrow!

I may misunderstand what you're saying, but is what you're saying that you checked the output of this and it looks correct? And that you'd like to have this merged as it currently stands? That's ok with me, I just need that clarification :-)

Reply:
Thank you!
Yes and no. The output of the PR as is is incorrect. 
I checked that it correctly put in **A** documentation for `create_triangulation` before starting this PR
but had not noticed that it was the wrong one.
The version on my laptop (to be rebased and pushed Thursday) puts in the correct documentation and 
does not have a recursion problem.

As is the indent fails as the doxygen version is to old, so no merging please!
Otherwise this will fail all coming PRs.

I did however spend the few minutes of rebasing #16024 to the current master (with merged #16067) 
and added the suppression of the copydoc warnings by removing them from the log.
So if the tests pass #16024 is the one to be merged next.

After that I will rebase this PR here and remove the sed commands from the YAML. 

Finally, #16013 can then be merged after rebase. 

And the doxygen problems of #15092 should be the same so the indent ubuntu version can then be updated as well.

Reply:
> But as this takes a bit of concentration I will likely do this on Thursday as I just took a quick break from preparing my defense tomorrow.

Good luck!